modules/gapi/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set(gapi_srcs
     src/backends/fluid/gfluidbuffer.cpp
     src/backends/fluid/gfluidbackend.cpp
     src/backends/fluid/gfluidimgproc.cpp
-    src/backends/fluid/gfluidimgproc_func.cpp
```

Comment:
Looks like this file is not removed from sources: src/backends/fluid/gfluidimgproc_func.cpp

Reply:
Yes, I forgot removing gfluidimgproc_func.cpp, thanks for finding this problem!

Reply:
Removing this needless file with my next PR #13157